roles/network_plugin/calico/templates/calico-cr.yml.j2

```diff
@@ -31,6 +36,15 @@ rules:
       - create
       - update
       - delete
+  - apiGroups: ["projectcalico.org/v3"]
```

Comment:
That doesn't seem necessary either. see https://github.com/projectcalico/calico/blob/master/manifests/calico.yaml

Reply:
calcicoctl needs this to modify its CR during the upgrade.
In our case the `calico-rr-id` in the Node object from this APIGroup